app/src/main/java/me/ivg2/foodapp/RecipeAdapter.java

```diff
@@ -110,7 +110,7 @@ public ViewHolder(View itemView) {
         public void onClick(View v) {
             int index = getAdapterPosition();
 
-            if (index != RecyclerView.NO_POSITION || index != -1) {
+            if (index != RecyclerView.NO_POSITION || index != -1 || recipes.get(index) == null) {
```

Comment:
Wait, shouldn't these be &&s?  All of these things should be true for the recipes.get to work.